modules/network-observability-resources-table.adoc

```diff
@@ -0,0 +1,28 @@
+//module included in the following assemblies:
+// network_observability/configuring_operator.adoc
+:_content-type: REFERENCE
+[id="network-observability-resources-table{context}"]
+= Resource recommendations
+The following table documents recommendations for resources depending on workload size. 
+
+[IMPORTANT]
+====
+These recommended scenarios are tailored to specific workloads and should serve only as a baseline from which adjustments can be made to accommodate your workload needs.
+====
+
+.Resource recommendations
+[options="header"]
+|============================================================================================================================
+|                                     | Extra small (10 Nodes) | Small (25 nodes)  | Medium (65 Nodes) ^[2]^ | Large (120 nodes) ^[2]^
+| *Worker Node vCPU and Memory*       | 4 vCPUs\| 16GiB Mem ^[1]^ | 16 vCPUs\| 64GiB Mem ^[1]^ | 16 vCPUs\| 64GiB Mem  ^[1]^  |16 vCPUs\| 64GiB Mem ^[1]^ 
+| *LokiStack Size*                    | 1x.extra-small         | 1x.small          | 1x.small           | 1x.medium
```

Comment:
`extra-small|small|small|medium` correct? No "large"? Just want to check because of the column headings.

Reply:
Sorry, I should have clarified: Does the LokiStack Size "1x.medium` entry definitely sit with the "Large (120 nodes) column" of would a LokiStack Size "1x.mlarge` entry (if it exists), be a better fit?

Reply:
Yes, the Large (120 nodes) correlates with the LokiStack size `1.xmedium`. This is based on QE performance testing data, you can see here: https://docs.google.com/document/d/1dKfMfHCtvXAh1-QE9vk08e3DWBjo7t2BSuNMcE1ECTQ/edit

Reply:
Thanks, Sarah. That makes sense. Feel free to mark this thread as Resolved.